tests/console/yast2_lan.pm

```diff
@@ -34,18 +34,71 @@ sub handle_dhcp_popup {
     }
 }
 
+
+sub check_etc_hosts_update {
+    my $looprun = 1;
+    assert_script_run "cat /etc/hosts";
+    until ($looprun == 3) {
+        script_run("yast2 lan; echo yast2-lan-status-\$? > /dev/$serialdev", 0);
+        assert_screen "yast2_lan";
+        send_key 'alt-i';    # edit NIC
+        assert_screen 'yast_ncurses_network_card_setup';
+        send_key 'alt-t';    # set to static ip
+        assert_screen 'yast_ncurses_set_static_ip';
+        send_key 'tab';
+        for (1 .. 15) { send_key "backspace" };    # delete existing IP if any
```

Comment:
can we try to replace it with 
```
send_key ctrl-a
send_key del
``` 
looks error prone.
@okurz, @rwx788  WDYT ?

Reply:
I would say this is the cheapest solution and proved to work. However, we replace it with `send_key_until_needle_match`. As we again have issues with typing even on 64bit, calling sequence of the keys is less stable.

Reply:
Hi, thanks for the tips, it looks like a good idea. But ctrl-a does not work on ncurses, so "send_key delete" either. 
So using send_key_until_needle_match makes the test a lot slower because we do an assert_screen every time we do "send_key 'backspace'".

Reply:
Another side point, we could have extracted method to clear text field as we have it already in >10 places. You already have assertion of what we have typed, so should be trivial to spot the case when not all the symbols were removed.